openeo/local/processing.py

```diff
@@ -59,4 +68,74 @@ def load_local_collection(*args, **kwargs):
 PROCESS_REGISTRY["load_collection"] = Process(
     spec=openeo_processes_dask.specs.load_collection, 
     implementation=load_local_collection,
+)
+
+def resample_cube_spatial_rioxarray(*args, **kwargs):
+    pretty_args = {k: type(v) for k, v in kwargs.items()}
+    _log.debug(f"Running process resample_cube_spatial")
+    _log.debug(f"kwargs: {pretty_args}")
+    _log.debug("-" * 80)
+
+    methods_list = {
```

Comment:
don't call a dictionary a list, that's quite confusing

also: where do these int values come from, is that ad-hoc, or is that defined somewhere?

Reply:
They come from the rioxarray - rasterio docs:

https://corteva.github.io/rioxarray/stable/rioxarray.html#rioxarray.raster_array.RasterArray.reproject_match
https://rasterio.readthedocs.io/en/stable/api/rasterio.enums.html#id0

Should I include the link in the code as a comment?

Reply:
Done

Reply:
I created this mapping since in openEO we have "near" instead of "nearest" and "cubicspline" instead of "cubic_spline"

Reply:
then I would just translate these minor differences, but let `rasterio.enums.Resampling` do the heavy lifting, e.g.
```python
method = {
    "near": "nearest",
    "cubicspline": "cubic_spline"
}.get(method, method)
try:
    resampling = rasterio.enums.Resampling[method]
except KeyError:
    ValueError(
            f'Selected resampling method not available.... select one of {[r.name for r in rasterio.enums.Resampling]}" ...
```

Reply:
Fine. However, if we raise that ValueError, we would list nearest and cubic_spline as allowed resampling methods, that if used in a process graph would make it invalid if there's some validation mechanism in place.

Reply:
@soxofaan I think that this is the last point open before being able to merge. Can we agree on my latest version?